blog/_posts/2024-02-29-fake-scala-courses.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: blog-detail
+post-type: blog
+by: Anatolii Kmetiuk, Scala Center
+title: Beware of fake online Scala courses
+---
+Lately, there has been an increased amount of fraud in the Scala community related to online courses. In this post, we mention some recent events that have been brought to our attention, and remind the community of trustworthy sources and ways to identify scams.
+
+## Recent events
+We are aware of at least the following incidents involving fraud:
+
+- Some bad actor was running a website under the domain scala dash language dot org, which was an exact copy of the official Scala's website, scala-lang.org. The scammers posed as an online recruiting firm and asked candidates applying for their "jobs" to complete Scala courses there, for a fee.
+- Another instance involved a Scala course hosted at graphy dot com (a platform where anyone can publish a course, not itself responsible for its users' actions and content). The course was not in any way impersonating the Scala Center or EPFL - in fact, their authors were completely anonymous, not even the certificate featured any identifying signs of an issuing person or organisation. However, despite being titled as a Data Science with Scala course, the course contained only very basic Scala content (think control structures and basic collections), and was priced at 200$ equivalent.
+
+Both instances are scam and, as of the date of this blog, are taken down. We've also forwarded the scala dash language dot org case to the legal department of EPFL, which will be taking appropriate legal action.
+
+## Trustworthy sources
+The Scala Center would like to use this opportunity to remind you of the only places on the Internet where you can find our official online courses are documented on the [official website](https://docs.scala-lang.org/online-courses.html#programming-reactive-systems). All of them offer a high-quality learning experience, and most are free to audit (i.e. without the final certificate or human involvement in grading).
+
+We would also like to remind the Scala community of the exhaustive list of domains from which you can find trustworthy information coming from the Scala organization:
+- [scala-lang.org](https://www.scala-lang.org/), notably including the following main subdomains:
+    - https://www.scala-lang.org/, the main website, which you are currently reading
+    - https://docs.scala-lang.org/, the documentation website
+- [scala-js.org](https://www.scala-js.org
+- [scala-native.org](https://scala-native.org/en/stable/) for the platform-specific dialects of Scala
```

Comment:
```suggestion
- [scala-js.org](https://www.scala-js.org)
- [scala-native.org](https://scala-native.org/en/stable/)
```
If you put it on two lines, the "for the platform-specific dialects" does not make sense ;)